docs/notebooks/.ipynb_checkpoints/WMD_tutorial-checkpoint.ipynb

```diff
@@ -0,0 +1,601 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Word Mover's Distance\n",
+    "\n",
+    "Word Mover's Distance (WMD) is a new and exciting method in machine learning that helps us measure the similarity of two sentences. WMD can be used for document retrieval, and has been shown to outperform many of the current state-of-the-art methods in $k$-nearest neighbors classification [1]. As we shall see, the benefit of WMD is that we can accurately assess the similarity of two documents, even when they have no words in common.\n",
+    "\n",
```

Comment:
Please give an overview of the post.
"WMD is used to find similar documents. For example, [opentable](http://tech.opentable.com/2015/08/11/navigating-themes-in-restaurant-reviews-with-word-movers-distance/) shows reviews from different angles on their restaurant page.  They use WMD to filter out similar ones. We show how to do it in Part 2 using WMDSimilarity class.  Part 1 is an optional introduction to the raw WMD distance."